dll/cpl/input/lang/ja-JP.rc

```diff
@@ -213,7 +213,8 @@ BEGIN
     IDS_PORTUGUESE_BRAZILIAN_ABNT_LAYOUT "ポルトガル語 (ブラジルABNT)"
     IDS_PORTUGUESE_BRAZIL_ABNT2_LAYOUT "ポルトガル語 (ブラジルABNT2)"
     IDS_PUNJABI_LAYOUT "パンジャブ語"
-    IDS_ROMANIAN_LAYOUT "ルーマニア語"
+    IDS_ROMANIAN_LEGACY_LAYOUT "ルーマニア語 (レガシー)"
+    IDS_ROMANIAN_STANDARD_LAYOUT "ルーマニア語 (標準)"
```

Comment:
I'd argue it's better to let other people translate the strings in their specific language accordingly. Just let them in English for now.

Reply:
Unless he also knows japanese.

Reply:
Leaving them in English is exactly what I did when I didn't had a model to follow on (haven't used Google Translate, in case you were wondering). Following models in such cases, is a practice I've noticed before on Romanian resources, and the results happened to be good too! :)

Reply:
These Japanese texts are correct.

Reply:
Thank you, Hirofumi!